CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic
 Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
This should be gone since we have no changes after this that are waiting to be released